### PR TITLE
revolutiontt.me

### PR DESCRIPTION
**Pull request by galeksandrp**

### CODE REVIEW

**Comment by J0WI** [comment]
I get a 403 on the root of this host.
Can you provide a valid test url?

File: src/chrome/content/rules/Revolutiontt.me.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<ruleset name="Revolutiontt.me">
+	<target host="revolutiontt.me" />
+	<target host="www.revolutiontt.me" />
```
Comment (J0WI) [inline src/chrome/content/rules/Revolutiontt.me.xml]: img.revolutiontt.me

**Comment by galeksandrp** [comment]
Only this urls is publicily available:
https://revolutiontt.me/login.php
https://revolutiontt.me/recover.php
https://revolutiontt.me/irchelp.php
https://m.revolutiontt.me/index.php

**Comment by J0WI** [comment]
Therefore `m.revolutiontt.me` should be added too.
But I get a 403 on all these urls.

**Comment by J0WI** [comment]
Worked with other IPs.

**Pull request merged**